localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -86,7 +86,7 @@ class Invocation:
 
 class ArchiveCode(metaclass=ABCMeta):
     @abstractmethod
-    def generate_presigned_url(self, endpoint_url):
+    def generate_presigned_url(self, endpoint_url: str | None = None):
```

Comment:
Small change here, to match the signature of the implementation subclasses (detected this as it was highlighted by the IDE..)